autopub/base.py

```diff
@@ -104,6 +104,11 @@ def dict_get(_dict, keys, default=None):
 GIT_USERNAME = dict_get(config, ["tool", "autopub", "git-username"])
 GIT_EMAIL = dict_get(config, ["tool", "autopub", "git-email"])
 
+# GitHub
+
+GITHUB_TOKEN = os.environ.get("GITHUB_TOKEN", None)
+APPEND_GITHUB_CONTRIBUTOR = dict_get(config, ['tool', 'autopub', 'append-github-contributor'], False)
```

Comment:
In the interest of consistency, could you change the single quotation marks to double quotation marks here?